global/ci-infra.yaml

```diff
@@ -0,0 +1,92 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  PYTHONWARNINGS: 'ignore:pkg_resources::'
+build_tool: colcon
+build_tool_test_args: '--event-handler console_stderr- console_cohesion+ --executor parallel --python-testing pytest --pytest-args -k "not test_spell_check" --packages-skip superflore'
```

Comment:
Is superflore being skipped because tests aren't passing on HEAD?

Reply:
That's correct. I think at least some of them may need a functioning container engine.

Reply:
cc @robwoolley since you have been getting superflore tests up and running.